stderr: Install for [ '@angular/cli@latest' ] failed with code 1

**Before submitting the bug report**
- [x] Make sure you enable DebugMode in vscode preferences and check the output channel Test Explorer Logs.
- [x] Make sure your project runs and 'ng test' properly runs all tests.

**Describe the bug**
After following the instruction in [Documentation](https://github.com/Raagh/angular-karma_test-explorer/blob/master/DOCUMENTATION.md), I open the Test Explorer.

I notice that `reload button` spins for sometime (5-8 secs) and then throws the below error in the Logs

**Logs**
```
[8:27:28 PM] INFO: Starting Angular test enviroment for project: app
[8:27:33 PM] INFO: Listening to AngularReporter events on port 9877
[8:28:24 PM] ERROR: stderr: npm
[8:28:24 PM] ERROR: stderr:  ERR! Unexpected end of JSON input while parsing near '...3j1ayaY62PYvLPwwuIB\n'

[8:28:24 PM] ERROR: stderr: 

[8:28:24 PM] ERROR: stderr: npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/libin/.npm/_logs/2021-02-10T14_58_24_037Z-debug.log

[8:28:24 PM] ERROR: stderr: Install for [ '@angular/cli@latest' ] failed with code 1
```

**Desktop (please complete the following information):**
- OS: OSX 10.13.6
- Angular Version - 10
- Node version - 12
- karma.conf.js path from the root folder you re opening in vscode.

--- a/src/core/angular/angular-process-configurator.ts
+++ b/src/core/angular/angular-process-configurator.ts
@@ -44,7 +44,7 @@ export class AngularProcessConfigurator {
       cliArgs = commonArgs;
       cliCommand = "ng";
     } else if (isAngularInstalledLocally) {
-      cliArgs = ["@angular/cli", ...commonArgs];
+      cliArgs = ["ng", ...commonArgs];
       cliCommand = "npx";
     } else {
       const error = "@angular/cli is not installed, install it and restart vscode";

